besu/src/main/java/org/hyperledger/besu/controller/BesuControllerBuilder.java

```diff
@@ -85,7 +85,7 @@
   protected PrivacyParameters privacyParameters;
   protected Path dataDirectory;
   protected Clock clock;
-  KeyPair nodeKeys;
+  NodeKey nodeKey;
```

Comment:
shouldn't this be `protected` or `private`?

Reply:
no - this is a builder, so this field has a setter.

Reply:
Looks like it could be protected